src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react'
```

Comment:
All Tooltip things have been refactored to TypeScript, along with latest/greatest HSDS: React conventions